tests/components/freebox/test_switch.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for the Freebox switches."""
+
+from unittest.mock import AsyncMock, Mock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN, SERVICE_TOGGLE
+from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.entity_registry as er
+
+from .common import setup_platform
+from .const import DATA_LAN_GET_PORT_FORWARDING_CONFIG_LIST
+
+
+async def test_port_forwarding(
+    hass: HomeAssistant, freezer: FrozenDateTimeFactory, router: Mock
+) -> None:
+    """Test port forwarding switch."""
+    await setup_platform(hass, SWITCH_DOMAIN)
+
+    registry = er.async_get(hass)
+    entity_id = "switch.port_forwarding_1"
+
+    entity = registry.async_get(entity_id)
+    assert entity.disabled_by == "integration"
+
+    # Simulate enabling the switch
+    registry.async_update_entity(entity_id, disabled_by=None)
+    await hass.config_entries.async_reload(entity.config_entry_id)
+    await hass.async_block_till_done()
```

Comment:
I would propose
1. have a test that tests that its disabled
2. apply the `entity_registry_enabled_by_default` fixture here, and then you don't have to enable and reload